.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - setup_remote_docker
       - add_ssh_keys:
           fingerprints:
-            - 2d:1f:7e:83:38:a2:f1:1c:89:86:c3:dd:a7:0e:32:2e
+            - SHA256:HDjXb+EIco/xPtCiblLoN2WDyjfRl8488OsY+JW0EoM
```

Comment:
Are we sure it should include the 'SHA256' prefix? it feels strange

Reply:
yes.
see example: https://circleci.com/docs/add-ssh-key/

Reply:
ok cool. Thanks.